pg_pong.py

```diff
@@ -0,0 +1,161 @@
+""" Trains an agent with (stochastic) Policy Gradients on Pong. Uses OpenAI Gym.
```

Comment:
Unfortunately renaming pg_pong means that I can't easily see the differences between the previous version and the new version. Can you break this PR up into two, such that the first simply renames pg-pong to pg_pong, and the second introduces your refactoring and unit test?

Reply:
The experimental branch is two commits ahead of the master branch. The first commit does the renaming. The second one contains all of the changes